youtube_dl/extractor/wrzuta.py

```diff
@@ -37,23 +38,57 @@ class WrzutaIE(InfoExtractor):
             'uploader_id': 'jolka85',
             'description': 'md5:2d2b6340f9188c8c4cd891580e481096',
         },
+    }, {
+        'url': 'http://rencz.wrzuta.pl/playlista/0pzHVJu0jAd/amy_winehouse',
+        'only_matching': True,
     }]
 
     def _real_extract(self, url):
         mobj = re.match(self._VALID_URL, url)
-        video_id = mobj.group('id')
+        extracted_id = mobj.group('id')
         typ = mobj.group('typ')
         uploader = mobj.group('uploader')
+        media_title = mobj.group('media_title')
 
-        webpage = self._download_webpage(url, video_id)
+        if typ == 'playlista':
```

Comment:
Got it. Looking at others extractors I got an impression that you want to keep all logic related to one site within single extractor. If creating a new one for extracting a playlist is allowed then it's good. I'll revert my changes done to wrzuta and create playlist extractor.